.github/workflows/buid_and_release.yml

```diff
@@ -4,8 +4,8 @@
 name: Build and release
 
 on:
-#  schedule:
-#    - cron: "0 0 * * 0"  # weekly on Sunday at midnight
+  schedule:
```

Comment:
Reactivated weekly cronjob now that I've made other changes to the action:
- Removed a file that was too big to be attached
- Removed part of the build process which was running out of memory

Also added `medgen-xrefs.robot.template.tsv` to the release.